### PR TITLE
Second PR to fabio.md

### PR DESCRIPTION
**Pull request by lh536**
This is my second PR made with gitbash. Hope everything is okay.

### CODE REVIEW

**Review by madhusudanbabar — CHANGES_REQUESTED** [review]
There's a typo
~imprvue~

**Comment by madhusudanbabar** [comment]
Your PR is fine, but this is how code reviews work

**Comment by lh536** [comment]
Thanks @madhusudanbabar and @johnjacobkenny! It's nice to see how code reviews works! Btw... sorry because of my typos.

**Comment by lh536** [comment]
I think now it's okay

File: firstpr/fabio.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
 #### Fabio Duarte
 I'm from Colombia.
 I'm a beginner with some experience in HTML, CSS and JavaScript. Lately I've using Tailwindcss.
```
Comment (UnevenCoder) [inline firstpr/fabio.md:3]: I've been using 

Instead of

I've using

**Comment by UnevenCoder** [comment]
Left a remark , after it all should be fixed 😀

**Review by UnevenCoder — APPROVED** [review]
Waiting for the changes 😃 so no language error is there

File: firstpr/fabio.md (hunk 1 above)
Comment (lh536) [reply firstpr/fabio.md:3]: Thank you!

**Comment by UnevenCoder** [comment]
All done .. just wait for the mods they will merge it as soon as possible

**Comment by johnjacobkenny** [comment]
@lh536 
Yea this is how there is a discussion back and forth to improve code 🙂

**Pull request merged**